### PR TITLE
[FIX] pyopenms enum for win

### PR DESCRIPTION
**Pull request by jpfeuffer**
This fixes it on Windows. Does it make sense?

### CODE REVIEW

**Comment by jpfeuffer** [comment]
Seems like you have to access it with pyopenms.RTUnit now (without RetentionTime). Even if I add back the wrap_attach.
Don't know if this is wanted.
I don't get why it compiles on Linux and Mac, though.

**Comment by hroest** [comment]
> Seems like you have to access it with pyopenms.RTUnit now (without RetentionTime). Even if I add back the wrap_attach.
> Don't know if this is wanted.

its not that bad, its just an issue of polluting the global pyopenms namespace with these small enums that should be in their own namespace / attached to a class

@jpfeuffer can you check whether adding the "wrap-attach" still works on Windows?

File: src/pyOpenMS/pxds/TargetedExperimentHelper.pxd [hunk 1]
```diff
@@ -6,26 +6,28 @@ from Map cimport *
...
-        PREDICTED,        # predicted by referenced software
-        HPINS,            # H-PINS "The de facto standard providing the retention times"
-        IRT,              # iRT retention time standard
-        UNKNOWN,          # no stored annotation
-        SIZE_OF_RTTYPE
+cdef extern from "<OpenMS/ANALYSIS/TARGETED/TargetedExperimentHelper.h>" namespace "OpenMS::TargetedExperimentHelper::RetentionTime::RTUnit":
+
+    ctypedef enum RTUnit "OpenMS::TargetedExperimentHelper::RetentionTime::RTUnit":
+            # wrap-attach:
+            #    RetentionTime
```
Comment (hroest) [inline src/pyOpenMS/pxds/TargetedExperimentHelper.pxd]: with this all tests pass again ...

**Comment by jpfeuffer** [comment]
Yes! I tried again today and with the wrap-attach it also works on windows. There must have been some old code cached somewhere. Will merge to see the effect on the nightlies.

**Pull request merged**